demos/JWST/S2_nirspec_fs_template.ecf

```diff
@@ -6,8 +6,8 @@ suffix                  rateints    # Data file suffix
 
 # Note: different instruments and modes will use different steps by default
 skip_wavecorr           True    # By default is skipped for TSO data
-skip_flat_field         True    # ***NOTE*** At the time the NIRSpec ERS Hackathon simulated data was created, this step did not work correctly and is by default turned off.
-skip_photom             True	# Recommended to skip to get better uncertainties out of Stage 3.
+skip_flat_field         False
```

Comment:
I think we want the default value to be `True` here, right?

Reply:
Ah, I'd figured you'd want it turned on since you were telling me that the step now works - I'll switch it back to be skipped

Reply:
Do you want it default to `True` for NIRSpec in the s2_meta.py file as well?

Reply:
The step is required when generating calibrated spectra, but I haven't tested it for standard reductions.  Let's keep the default value as `True` everywhere and inform people in the comments that it should be set to `False` when generating calibrated spectra.